mbofui/app/index.html

```diff
@@ -85,110 +85,111 @@
+
+      <!-- My Events Modal -->
+      <div class="modal fade" id="myEventsModal" tabindex="-1" role="dialog" aria-labelledby="myEventsLabel">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title" id="myEventsLabel">My Events</h4>
+            </div>
+            <div class="modal-body">
+              <div ng-include="'views/myevents.html'">
+              </div>
+            </div>
+          </div>
         </div>
-        <div class="modal-body">
-          <div ng-include="'views/coords-lookup.html'"></div>
+      </div>
+
+      <!-- Coords LookUp Modal -->
+      <div class="modal fade" id="coordsLookUpModal" tabindex="-1" role="dialog" aria-labelledby="coordsLookUpLabel">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title" id="coordsLookUpLabel">Specify Address</h4>
+            </div>
+            <div class="modal-body">
+              <div ng-include="'views/coords-lookup.html'"></div>
+            </div>
+          </div>
         </div>
       </div>
     </div>
-  </div>
-
-  <!-- libraries -->
-
-  <script src="bower_components/leaflet/dist/leaflet.js"></script>
-  <script src="bower_components/angular/angular.js"></script>
-  <script src="bower_components/angular-simple-logger/dist/angular-simple-logger.min.js">
-  </script>
-  <script src="../bower_components/leaflet-plugins/layer/tile/Google.js"></script>
-  <script src="bower_components/jquery/jquery.min.js"></script>
-  <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
-  <script src="bower_components/leaflet.markercluster/dist/leaflet.markercluster-src.js"></script>
-  <script src="bower_components/ui-leaflet/dist/ui-leaflet.min.js"></script>
-  <script src="bower_components/Leaflet.awesome-markers/dist/leaflet.awesome-markers.min.js"></script>
-  <script src="bower_components/underscore/underscore-min.js"></script>
-  <script src="bower_components/moment/min/moment.min.js"></script>
-  <script src="bower_components/webshim/js-webshim/minified/polyfiller.js"></script>
-  <script src="//maps.googleapis.com/maps/api/js?key=AIzaSyBJfRs5m2aIl3cKjMy0es9Fh6-EJeXpP4k"></script>
-
-  <!-- grunt concatenated app files -->
-  <script src="js/dist/all.min.js"></script>
+    <!-- libraries -->
+
+    <script src="bower_components/leaflet/dist/leaflet.js"></script>
+    <script src="bower_components/angular/angular.js"></script>
+    <script src="bower_components/angular-simple-logger/dist/angular-simple-logger.min.js">
+    </script>
```

Comment:
Yeah. I added an outer div to carry the role="main" so I indented everything it contained.